cmd/alert-gateway/initial/timer.go

```diff
@@ -271,105 +116,32 @@ func Filter(alerts map[int64][]Record, maxCount map[int64]int) map[string][]comm
 		for _, element := range Cache[planId.PlanId] {
 			if element.IsValid() && element.IsOnDuty() {
 				if maxCount[key] >= element.Start {
-					if _, ok := common.RuleCount[[2]int64{key, int64(element.Start)}]; !ok {
-						NewRuleCount[[2]int64{key, int64(element.Start)}] = -1
+					k := [2]int64{key, int64(element.Start)}
+					if _, ok := common.RuleCount[k]; !ok {
+						NewRuleCount[k] = -1
 					} else {
-						NewRuleCount[[2]int64{key, int64(element.Start)}] = common.RuleCount[[2]int64{key, int64(element.Start)}]
+						NewRuleCount[k] = common.RuleCount[k]
 					}
-					NewRuleCount[[2]int64{key, int64(element.Start)}] += 1
+					NewRuleCount[k] += 1
 
-					if NewRuleCount[[2]int64{key, int64(element.Start)}]%int64(element.Period) == 0 {
+					if NewRuleCount[k]%int64(element.Period) == 0 {
```

Comment:
For this part, we use  k := [2]int64{key, int64(element.Start)} instead of multiple  [2]int64{key, int64(element.Start)}  so that the code will be simpler.